impl/spanner/batch/batch_test.go

```diff
@@ -29,13 +29,13 @@ import (
 	tpb "github.com/google/trillian"
 )
 
-func NewForTest(ctx context.Context, t *testing.T, dirID string) (*Table, func()) {
+func NewForTest(ctx context.Context, t *testing.T, dirID string) sequencer.Batcher {
```

Comment:
In the interests of one-thing-at-a-time refactoring, can you change this back to *Table ?